tests/test_check_redcap_event.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+
+from nacc.redcap2nacc import check_redcap_event
+
+
+class option():
+    lbd = False
+    ftld = False
+    csf = False
+    np = False
+    m = False
+    ivp = False
+    fvp = False
+
+
+class TestRedcapEvent(unittest.TestCase):
+    '''
+    These tests are meant to ensure that the test_redcap_event function is
```

Comment:
Did you mean `check_redcap_event`?
```suggestion
    These tests are meant to ensure that the check_redcap_event function is
```

Reply:
good catch, thank you.